tests/test_streamingclient.py

```diff
@@ -100,3 +100,10 @@ def test_end(self, mock_streaming_client):
         mock_streaming_client.end()
 
         mock_streaming_client.client.abort.assert_called_once_with()
+
+
+def validate_query_parameters(called_url, query_dict):
```

Comment:
This is fine here for now but we should move it to a utils file when we update all the client's to use `urlunparse`